src/main/webapp/components/InstitutionalSource.js

```diff
@@ -25,8 +25,8 @@ export const InstitutionalSource = hh(class InstitutionalSource extends Componen
   addInstitutionalSources() {
     if (this.props.edit) {
       // Only for edit / review
-      if ((this.props.institutionalSources[0].future.name !== '' && this.props.institutionalSources[0].future.country !== '')
-        || this.props.institutionalSources[0].current.name && this.props.institutionalSources[0].current.country ) {
+      if (this.props.institutionalSources[0] && (this.props.institutionalSources[0].future.name !== '' && this.props.institutionalSources[0].future.country !== '')
+        || this.props.institutionalSources[0] && (this.props.institutionalSources[0].current.name && this.props.institutionalSources[0].current.country )) {

```

Comment:
This is a boatload of property checking ... I count 6 in this one line. If we find ourselves needing this, we should be looking for a better solution. Maybe a combination of lodash get and isEmpty would make this easier to read and reason about. 

* https://lodash.com/docs/4.17.15#get
* https://lodash.com/docs/4.17.15#isEmpty

Reply:
I've changed this as you suggested. used Get and IsEmpty from lodash.